src/gsy_e/gsy_e_core/simulation/simulation.py

```diff
@@ -422,14 +425,21 @@ def _time_since_start(self) -> Duration:
         return current_time - self.config.start_date
 
     def _deactivate_areas(self, area: "AreaBase"):
-        pass
+        if area.strategy:
+            area.strategy.deactivate()
+        for child in area.children:
+            self._deactivate_areas(child)
 
     def _execute_simulation(
             self, slot_resume: int, _tick_resume: int, console: NonBlockingConsole = None) -> None:
         slot_count, slot_resume = (
             self._time.calc_resume_slot_and_count_realtime(
                 self.config, slot_resume))
 
+        if self.config.external_connection_enabled:
+            self.config.external_redis_communicator.sub_to_aggregator()
+            self.config.external_redis_communicator.start_communication()
+
```

Comment:
I think that this is also not needed, if these were performed in the constructor of the external_redis_communicator.

Reply:
Let me try this. I only copied this from the main simulation class. There must be a reason why we did not put these in the constructor of the `ExternalConnectionCommunicator` class.

Reply:
@spyrostz we have a problem with this change. The connection to redis might not be initiated in this when calling these methods in the init of the external_redis_communicator [here](https://github.com/gridsingularity/gsy-e/pull/1559/commits/0d2cbfcb410817f89992821aed22687ae80b80af#diff-dacdd5512566d09ead79d7866dc73d81fffa3907abad2f0880ddb06c8af5d343R158) (have a look at the failing tests, you can reproduce these locally by turning off the redis server)
This is probably why it was pulled out of there into the simulation class. I propose to add an `activate` method that is called after the init (from the simulation class) which is the same implementation as before, only grouped into a method. What do you thin?

Reply:
@hannesdiedrich too bad to hear! I agree with your suggestion, and I think that it is good because it will separate the parts of the code that need to be part of the activate process. Thanks again!

Reply:
I did that [cee9417](https://github.com/gridsingularity/gsy-e/pull/1559/commits/cee9417511d93ec337c3e32943889dc3aa19c5a4)